src/BlueprintParser.h

```diff
@@ -357,14 +357,37 @@ namespace snowcrash {
                 return;
             }
 
-            // Otherwise, add the respective entries to the tables
-            if (typeDefinition.typeSpecification.name.base != mson::UndefinedTypeName) {
+            mson::BaseTypeName baseTypeName = typeDefinition.typeSpecification.name.base;
+
+            // Initialize an entry in the dependency table if not found
+            if (pd.namedTypeDependencyTable.find(identifier) != pd.namedTypeDependencyTable.end()) {
```

Comment:
Oh lost in the diff. It is "just" something over 100 (but thats still too long) – OK to leave for now tho.